Tests/MapboxMapsTests/Gestures/GestureHandlers/DoubleTapToZoomInGestureHandlerTests.swift

```diff
@@ -61,4 +61,16 @@ final class DoubleTapToZoomInGestureHandlerTests: XCTestCase {
         easeToCompletion(.end)
         XCTAssertEqual(delegate.animationEndedStub.parameters, [.doubleTapToZoomIn])
     }
+
+    func testFocalPoint() {
+        let focalPoint = CGPoint(x: 1000, y: 1000)
+        gestureHandler.focalPoint = focalPoint
+        gestureRecognizer.getStateStub.defaultReturnValue = .recognized
+        mapboxMap.cameraState = .random()
```

Comment:
Is setting the cameraState to random relevant to this test?

Reply:
Not really, a copy-paste thing most likely.